Update assist docs

klizhentas opened the pull request.

klizhentas committed:
Update assist docs

jakule commented on docs/pages/ai-assist.mdx:
```diff
@@ -83,20 +83,20 @@ Do not share your secret API keys in publicly accessible areas.
 
 To enable Teleport Assist, you need to provide your OpenAI API key. On each
-Proxy Service host, perform the following actions:
+Auth Service host, perform the following actions:
```
This is not exactly true. Right now OpenAI key is required in Auth and Proxy.

klizhentas replied:
so you need it in both `auth_service` and `proxy_service`?

jakule replied:
Yes, added to this PR.

jakule committed:
Update AI Assist documentation for multiple hosts

The AI Assist documentation was updated to clarify the configuration process for both Proxy and Auth Service hosts.

jakule commented:
@ptgott @alexfornuto Friendly ping

ptgott commented on docs/pages/ai-assist.mdx:
````diff
@@ -83,20 +83,31 @@ Do not share your secret API keys in publicly accessible areas.
+      api_token_path: /etc/teleport/openai_key
+```
+
+If the host is running Proxy service, add the following section:
````
```suggestion
If the host is running the Proxy Service, add the following section:
```

ptgott reviewed (approved):
Approved with minor suggestions

jakule committed:
Apply suggestions from code review

Co-authored-by: Paul Gottschling <paul.gottschling@goteleport.com>
